### PR TITLE
Switch to topkg

### PR DESCRIPTION
**Pull request by samoht**
Signed-off-by: Thomas Gazagnaire <thomas@gazagnaire.org>

### CODE REVIEW

File: _tags [hunk 1]
```diff
@@ -0,0 +1,40 @@
+true : bin_annot, safe_string, package(bytes)
```
Comment (avsm) [inline _tags]: could you add `debug` here as well?

Comment (avsm) [inline _tags]: and move all the `package()` declarations to the same part of the tags file?

File: src/irmin-io/ir_io.mli [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module Sync: Git.Sync.IO
```
Comment (avsm) [inline src/irmin-io/ir_io.mli]: missing license?

File: _tags (hunk 1 above)
Comment (dbuenzli) [inline _tags]: Regarding `debug` feed back about https://github.com/dbuenzli/topkg/issues/54 is welcome.

File: src/irmin-io/ir_io.mli (hunk 2 above)
Comment (samoht) [inline src/irmin-io/ir_io.mli]: The apache license says that you should only add a license file at the root of the repo. I'm happy to add license headers in the file too if that's necessary.

**Pull request merged**